luvit-loader.lua

```diff
@@ -183,7 +183,7 @@ local function loader(dir, path, bundleOnly)
          try(pathJoin(dir, "libs", path)) then
         break
       end
-      if #dir < 2 then
+      if dir == pathJoin(dir, "..") then
```

Comment:
See [this commit message](https://github.com/luvit/lit/pull/191/commits/0c2bdc984a93d499417b5c8109e2cf9815eb21df):

> Fix infinite loop when a module is not found on Windows
>  - Windows root is 2 characters, so the #dir < 2 check would fail over and over (pathJoin("C:", "..") = "C:")